runtime/near-vm-runner/src/cache.rs

```diff
@@ -111,12 +117,12 @@ impl fmt::Debug for MockCompiledContractCache {
 #[cfg(not(feature = "no_cache"))]
 const CACHE_SIZE: usize = 128;
```

Comment:
Thias *might* give unused if both are disabled?

Reply:
There are a fair amount of unused lints when none of the implementations are enabled, including for this item. I'm not entirely sure it is worth the effort to go and clean up everything here, as I only really needed this for some hacking with aurora contracts.

I don't anticipate disabling all the runtimes will be all that useful thing to need.